sematic/utils/exceptions.py

```diff
@@ -0,0 +1,23 @@
+# Standard Library
+import sys
+import traceback
+
+# Sematic
+from sematic.abstract_calculator import CalculatorError
+
+
+def format_exception_for_run() -> str:
```

Comment:
Naming nit: `format_exception`.

Reply:
I like the "for_run" because this formatting is really only desired if you are formatting the exception for display in a run. For printing to Sematic logs, for instance, you wouldn't want to skip the parts of the stack trace from Sematic code.